halo/evmstaking/keeper/keeper.go

```diff
@@ -335,6 +382,10 @@ func (k Keeper) createAccIfNone(ctx context.Context, addr sdk.AccAddress) {
 	}
 }
 
+func (k Keeper) hasAccount(ctx context.Context, addr sdk.AccAddress) bool {
+	return k.aKeeper.HasAccount(ctx, addr)
+}
+
```

Comment:
not a fan of functions that just call another function, it just obfuscates what is actually happening. Why not just call the actual function directly?